tests/interpreters/test_riscv_emulator.py

```diff
@@ -30,22 +30,24 @@ def body():
             riscv.CustomAssemblyInstructionOp(
                 "print", inputs=[forty_two], result_types=[]
             )
+            riscv.ReturnOp()
 
         riscv_func.FuncOp("main", body, ((), ()))
 
     RISCVRegisterAllocation().apply(ctx, module)
 
     code = riscv.riscv_code(module)
 
+    # TODO: fix the adding of .text .globl main
```

Comment:
Can you please add a `riscv.DirectiveOp(".globl", ".main")`? Should be fairly quick